test/db/formats/dex

```diff
@@ -216,35 +216,35 @@ EXPECT=<<EOF
 0x1000004a1 8 str.world
 0x1000004ab 8 str.worldVar
 0x1000004b5 9 str.worldVar2
-0x1000004f0 0 sym.LHello_.localVar2
-0x1000004f2 0 sym.LHello_.localVar
-0x100000506 0 sym.LWorld_.worldVar
-0x100000508 0 sym.LWorld_.worldVar2
+0x1000004f0 0 sym.Hello.localVar2
+0x1000004f2 0 sym.Hello.localVar
+0x100000506 0 sym.World.worldVar
+0x100000508 0 sym.World.worldVar2
 0x8000000000 14 section.reloc_targets
-0x8000000000 2 sym.Ljava_io_PrintStream_.println_Ljava_lang_String__V
-0x8000000002 2 sym.Ljava_lang_Integer_._init__I_V
-0x8000000004 2 sym.Ljava_lang_Integer_.toString__Ljava_lang_String
-0x8000000006 2 sym.Ljava_lang_Object_._init___V
-0x8000000008 2 sym.Ljava_lang_StringBuilder_._init___V
-0x800000000a 2 sym.Ljava_lang_StringBuilder_.append_Ljava_lang_String__Ljava_lang_StringBuilder
-0x800000000c 2 sym.Ljava_lang_StringBuilder_.toString__Ljava_lang_String
+0x8000000000 2 sym.java.io.PrintStream.void_println_String
+0x8000000002 2 sym.Integer.void__init__int
+0x8000000004 2 sym.Integer.String_toString
+0x8000000006 2 sym.Object.void__init
+0x8000000008 2 sym.StringBuilder.void__init
+0x800000000a 2 sym.StringBuilder.StringBuilder_append_String
+0x800000000c 2 sym.StringBuilder.String_toString
 EOF
 RUN
 
 NAME=DEX HelloWorld.dex imports (ii command)
 FILE=bins/dex/HelloWorld.dex
 CMDS=ii
 EXPECT=<<EOF
-nth vaddr      bind type  lib       name                                       
--------------------------------------------------------------------------------
-0   0x00000000 WEAK FIELD java.lang System.out
-1   ---------- WEAK FUNC  java.io   void PrintStream.println(String)
-2   ---------- WEAK FUNC  java.lang void Integer.<init>(int)
-3   ---------- WEAK FUNC  java.lang String Integer.toString()
-4   ---------- WEAK FUNC  java.lang void Object.<init>()
-5   ---------- WEAK FUNC  java.lang void StringBuilder.<init>()
-6   ---------- WEAK FUNC  java.lang StringBuilder StringBuilder.append(String)
-7   ---------- WEAK FUNC  java.lang String StringBuilder.toString()
+nth vaddr      bind type  lib                     name                                       
+---------------------------------------------------------------------------------------------
+0   0x00000000 WEAK FIELD java.lang.System        System.out
```

Comment:
Isn't this an unnecessary duplication of the name?

Reply:
no, is a dup due how the fields are created for the table.